### ISSUE TITLE
[Question] Can one ueransim(gnb) support multi ueransim(UE) ?

### ISSUE THREAD

**Issue by Evanlin1224** [aligungr/UERANSIM#315]
Hello, @aligungr 
I want to implement two or three UEs through one gnb to connect open5gs core. And i want to separate UEs ,like UE1,UE2,UE3....
Can UERANSIM achieve or support that multi UEs connect one gnb at the same time?

**Comment by aligungr** [comment]
Hello @Evanlin1224 

Yes, of course. You can use multiple UEs over same gNB. Please see wiki page, and let me know if you have a question

**Comment by Evanlin1224** [comment]
Sorry @aligungr ~
maybe i don't described in details. My multi UEs mean that UE1 (build in VM1)、UE2(build in VM2)、UE3(build in VM3), they connect gNB(build in VM4) at the same time.

**Comment by aligungr** [comment]
Yes, as long as you set the IP addresses in the config files correctly you can surely use them in separate machine. There is absolutely no limitation about this.

This maybe helpful:

https://github.com/s5uishida/open5gs_5gc_ueransim_sample_config

**Comment by Evanlin1224** [comment]
@aligungr  Thanks you so much!
I will try it and i will let you know ,if i have some problem!
